include/c/sk_types.h

```diff
@@ -472,6 +473,14 @@ typedef enum {
     HEIF_SK_ENCODED_FORMAT,
 } sk_encoded_image_format_t;
 
+/**
+ *  Enum describing selection policy.
+ */
+typedef enum {
+    kPreferStillImage,
+    kPreferAnimation,
+} sk_codec_selection_policy_t;
```

Comment:
The generator may stumble on this so you may have to go the all caps with the suffix:

PREFER_STILL_IMAGE_SK_CODEC_SELECTION_POLICY
PREFER_ANIMATION_SK_CODEC_SELECTION_POLICY

In fact it would be better to do this to be consistent with the other APIs as I know there are users of the C API for other binding libraries.

Reply:
> The only thing left is the enum added to sk_enums.cpp

like this?

```c
// sk_codec_selection_policy_t
static_assert ((int)SkCodec::SelectionPolicy::kPreferStillImage   == (int)PREFER_STILL_IMAGE_SK_CODEC_SELECTION_POLICY,   ASSERT_MSG(SkCodec::SelectionPolicy, sk_codec_selection_policy_t));
static_assert ((int)SkCodec::SelectionPolicy::kPreferAnimation    == (int)PREFER_ANIMATION_SK_CODEC_SELECTION_POLICY,     ASSERT_MSG(SkCodec::SelectionPolicy, sk_codec_selection_policy_t));
```